src/methods/using_methods/hard/q9/Tickets.java

```diff
@@ -0,0 +1,35 @@
+package methods.using_methods.hard.q9;
+
+/*
+Write a Java Program called Tickets that asks users the number of people getting a ticket for a trip.
```

Comment:
Change question text to
`
Write a Java Program called Tickets that asks users the number of people getting a ticket for a trip. Your program should have a method called tripCostCalc that accepts the number of people as an input parameter and returns the total cost of the tickets. For two or fewer, the cost of tickets is $5.00 per person; for 3-5 people, the price is $4.50 per person; for 6-10 people, the price is $4.00 per person; for 11 or more people, the price is $3.00 per person. Finally, print the total cost of the trip. 
`

Reply:
We should also maybe add to print 0 if the number of tickets entered is less than 0